style.css

```diff
@@ -0,0 +1,1125 @@
+    display: grid;
+    grid-template-columns: auto auto auto;
+    gap: 2%;
+    margin-left: 10%;
+    margin-right: 10%;
+    margin-bottom: 130px;
+  }
+
+  .nothing {
+    border: 1px solid black;
+    height: 435px;
+    background-image: url("Images/project-button-desktop.svg");
+    background-size: cover;
+  }
+
+  .project-button-desktop {
+    padding: 12px;
+    background-color: #f55800;
+    text-align: center;
+    justify-items: center;
+    width: 100%;
+    border: none;
+    color: #fff;
+    font-family: "Inter", sans-serif;
+    font-size: 17px;
+    font-weight: 700;
+    line-height: 24px;
+    position: relative;
+    top: 387px;
+  }
+
+  .project-button-desktop:hover {
+    cursor: pointer;
+    background-color: #fff5e1;
+    border: 1px solid #f55800;
+    color: #f55800;
+  }
+
+  .profesion-desktop {
+    height: 435px;
+    background-image: url("images/data-dashboard-desktop.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .heading-content-desktop {
+    margin-left: 16px;
+    margin-top: 175px;
+    margin-right: 16px;
+    margin-bottom: 0;
+    font-family: "Crete Round", serif;
+    font-size: 32px;
+    font-weight: 400;
+    line-height: 44px;
+    color: #fff;
+  }
+
+  .p-content-desktop {
+    margin-left: 16px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 17px;
+    line-height: 24px;
+    color: #fff;
+    margin-top: 10px;
+  }
+
+  .p-language-desktop {
+    display: flex;
+    margin-left: 16px;
+    margin-bottom: 16px;
+    gap: 12px;
+    margin-right: 24px;
+    padding: 0;
+    margin-top: 16px;
+  }
+
+  .block-two-desktop {
+    text-decoration: none;
+    list-style-type: none;
+    color: #fff;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    padding: 10px 12px;
+    background: rgba(255, 255, 255, 0.24);
+  }
+
+  .profesion-desktop3 {
+    height: 435px;
+    background-image: url("images/website-portfolio-desktop.svg");
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
+
+  .heading-content-desktop3 {
+    margin-top: 221px;
+    font-family: "Crete Round", serif;
+    font-size: 32px;
+    font-weight: 400;
+    line-height: 44px;
+    color: #fff;
+    margin-bottom: 0;
+    margin-left: 16px;
+  }
+
+  .profesion-desktop4 {
+    height: 435px;
+    background-image: url("images/professionalandprinting.svg");
+    background-repeat: no-repeat;
+  }
+
+  .heading-content-desktop4 {
+    margin-top: 175px;
+    font-family: "Crete Round", serif;
+    font-size: 32px;
+    font-weight: 400;
+    line-height: 44px;
+    margin-left: 16px;
+    color: #fff;
+    margin-bottom: 0;
+  }
+
+  .block-two-desktop:hover {
+    background-color: #fff5e1;
+    color: #f55800;
+  }
+
+  .project-cards {
+    display: none;
+  }
+
+  .about {
+    background-image: url("images/aboutmebackgrounddesk.svg");
```

Comment:
- [ ] Kindly align this background image properly according to the given template. 👍